cinema/views.py

```diff
@@ -48,6 +93,37 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    def apply_queryset_filters(self, queryset: QuerySet):
+        movie_id = self.request.query_params.get("movie")
+        date = self.request.query_params.get("date")
+
+        if movie_id:
+            queryset = queryset.filter(movie__id=int(movie_id))
+
+        if date:
+            queryset = queryset.filter(show_time=date)
```

Comment:
Does it work like in the example when you specify only the date in the filter?
`GET /api/cinema/movie_sessions/?date=2024-10-08&movie=1`

Reply:
No, it doesn't. But I have a more interesting question. Why does it pass tests?

Reply:
I do not know. I tested it manually :)

Reply:
Alright, I have the answer.
![image](https://user-images.githubusercontent.com/108597725/202857282-d3a941ea-f68f-45b5-a32a-05c392f11f21.png)

Reply:
Nice